src/components/components/CountDown.js

```diff
@@ -39,6 +39,40 @@ export const getCountDisplay = (
   )}`;
 };
 
+export const getCountDisplayToHours = (
+  countDown,
+  includeWords = true,
+  leadingZeros = false
+) => {
+  const addLeadingZeros = value => {
+    let valueS = String(value);
+    while (valueS.length < 2) {
+      valueS = "0" + valueS;
+    }
+    return valueS;
+  };
+  const displayTime = (unit, appendage = true) =>
+    countDown[unit] > 0
```

Comment:
It would probably be easier to read this if we were to this into a variable `countDown[unit]` or something. But not something I'm asking you to change. Just this code isn't super easy to read.

Reply:
I didn't change this, but sure